apps/server/src/shared/repo/school/legacy-school.repo.integration.spec.ts

```diff
@@ -114,7 +114,7 @@ describe('LegacySchoolRepo', () => {
 
 	describe('findByExternalId', () => {
 		it('should find school by external ID', async () => {
-			const system: SystemEntity = systemFactory.buildWithId();
+			const system: SystemEntity = systemEntityFactory.buildWithId();
```

Comment:
You are using the same entity factory for both repositories.. why we have two repositories if they are using the same entity?